src/NHibernate/NHibernate.csproj

```diff
@@ -1,42 +1,72 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../../build-common/NHibernate.props" />
+
   <PropertyGroup>
-    <Description>An object persistence library for relational databases.</Description>
+    <Description>NHibernate is a mature, open source object-relational mapper for the .NET framework. It is actively developed, fully featured and used in thousands of successful projects.</Description>
+    <PackageTags>ORM; O/RM; DataBase; DAL; ObjectRelationalMapping; NHibernate; ADO.Net; Core</PackageTags>
+
     <TargetFramework>net461</TargetFramework>
+    <DisableImplicitPackageTargetFallback>true</DisableImplicitPackageTargetFallback>
     <NoWarn>$(NoWarn);3001;3002;3003;3005;1591</NoWarn>
     <SignAssembly>True</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <AssemblyOriginatorKeyFile>..\NHibernate.snk</AssemblyOriginatorKeyFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
+
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
     <TreatSpecificWarningsAsErrors />
   </PropertyGroup>
+
   <ItemGroup>
     <None Remove="**\*.g" />
     <None Remove="**\*.xsd" />
   </ItemGroup>
+
   <ItemGroup>
     <Antlr3 Include="**\*.g" />
   </ItemGroup>
+
   <ItemGroup>
     <EmbeddedResource Include="**\*.xsd" />
   </ItemGroup>
+
   <ItemGroup>
     <None Include="..\NHibernate.snk" Link="NHibernate.snk" />
   </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="Antlr3.CodeGenerator" Version="3.5.2-beta1">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Antlr3.Runtime" Version="3.5.1" />
-    <PackageReference Include="iesi.collections" Version="4.0.2" />
-    <PackageReference Include="Remotion.Linq" Version="2.1.2" />
-    <PackageReference Include="Remotion.Linq.EagerFetching" Version="2.1.0" />
+    <PackageReference Include="Antlr3.Runtime" Version="[3.5.1, 4.0)" />
```

Comment:
I doubt there will be v4 of Antlr3

Reply:
Excluding v4 was how the previous `.nuspec` had it.  It doesn't hurt to keep it.